WordPress/Docs/Security/PluginMenuSlugStandard.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0"?>
+<documentation xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="https://phpcsstandards.github.io/PHPCSDevTools/phpcsdocs.xsd"
+    title="Plugin Menu Slug"
+    >
+    <standard>
+    <![CDATA[
+    Wordpress functions that can be used to add pages to the WP Admin menu should not include `__FILE__` for the menu slug (or parent menu slug) parameter to avoid revealing system paths.
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: Slug does not include `__FILE__`.">
+        <![CDATA[
+add_menu_page(
+    'My Plugin Main Page',
+    'My Plugin',
+    'manage_options',
+    <em>'my-plugin-main'</em>,
+    'my_plugin_main_page'
+);
+
+add_submenu_page(
+    <em>'my_plugin_main_page'</em>,
+    'My Plugin Subpage',
+    'Subpage',
+    'manage_options',
+    'my-plugin-subpage',
+    'my_plugin_subpage'
+);
+        ]]>
+        </code>
+        <code title="Invalid: Slug includes `__FILE__`.">
+        <![CDATA[
+add_menu_page(
+    'My Plugin Main Page',
+    'My Plugin',
+    'manage_options',
+    <em>__FILE__ . </em>'my-plugin-subpage',
```

Comment:
I apologize if my original comment was unclear. What I had in mind was to keep one example with just `__FILE__`, and since you were going to add another example using `add_submenu_page()`, use it as an opportunity to show that the sniff is triggered as well when `__FILE__` is concatenated with something else.

So, I would suggest reverting the change in this like to keep it as it was with just `__FILE__`. What do you think?

```suggestion
    <em>__FILE__</em>,
```